src/Microsoft.Sbom.Api/Output/Telemetry/TelemetryRecorder.cs

```diff
@@ -90,6 +151,30 @@ public void RecordSBOMFormat(ManifestInfo manifestInfo, string sbomFilePath)
         this.sbomFormats[manifestInfo] = sbomFilePath;
     }
 
+    /// <summary>
+    /// Write telemetry object to the telemetryFilePath.
+    /// </summary>
+    /// <param name="telemetry">The telemetry object to be written to the file.</param>
+    /// /// <param name="telemetryFilePath">The file path we want to write the telemetry to.</param>
+    public async Task RecordToFile(SBOMTelemetry telemetry, string telemetryFilePath)
```

Comment:
does this method need to be public? Or can it be private?

Reply:
Only used inside this class. Changed to private